hooks/post-process.js

```diff
@@ -0,0 +1,22 @@
+// vim: set ts=2 sw=2 sts=2 expandtab :
+const fs = require('fs');
+const path = require('path');
+const _ = require('lodash');
+
+module.exports = register => {
+  register('generate:after', generator => {
+    const asyncapi = generator.asyncapi;
+    const info = asyncapi.info();
+    //let package = generator.templateParams['javaPackage'];
```

Comment:
🚜

Reply:
Was it just the commented-out code here that was the problem? (I'll remove line 9 too as info isn't being used.)

Reply:
Yes, it was just about the commented code.